plugins/vppv2/aclplugin/vppcalls/interfaces_vppcalls.go

```diff
@@ -192,6 +207,9 @@ func (h *ACLVppHandler) DeleteMACIPACLFromInterface(aclIndex uint32, ifName stri
 	if err != nil {
 		return fmt.Errorf("failed to delete interface %d from MACIP ACL (L2) %d: %v", ifIdx, aclIndex, err)
 	}
+	if reply.Retval != 0 {
```

Comment:
These were excluded intentionally. The GoVPP's `ReceiveReply` will return error if it's non-nil. Please remove it and I will cleanup all the other vppcalls to remove it there as well.

Reply:
OK, will do